bedrock/firefox/templates/firefox/os/devices-panasonic-tv.html

```diff
@@ -49,7 +49,13 @@
           <a id="device-nav-smartphones" href="#smartphones">{{ _('Smartphones') }}</a>
         </li>
         <li>
-          <a id="device-nav-tvs" href="#tvs">{{ _('Televisions') }}</a>
+          <a id="device-nav-tvs" href="#tvs">
+          {% if l10n_has_tag('fxos_devices_panasonic_tv_update_1') %}
```

Comment:
I wonder if we can get away with `LANG.startswith('en-')` and avoid the fourth update to this page in a week.